helm_deploy/values-dev.yaml

```diff
@@ -1,30 +1,11 @@
-
-minReplicaCount: 1
-maxReplicaCount: 2
-
-springProfiles: sqs-read
-
-image:
-  repository: quay.io/hmpps/court-list-splitter
-  tag: latest
-  port: 8080
```

Comment:
I removed values that are already held in Values.yaml, to remove repetition as they were repeated for each environment.